### PR TITLE
Update Contact+About+Legals  style.css

### PR DESCRIPTION
**Pull request by Right-Noe**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -2587,6 +2594,39 @@ a.next.page-numbers:before {
...
 
+.dark-mode .contact h4.wp-block-heading, 
+.dark-mode .contact ul.no-bullets li a:hover {
+  color: var(--theme-secondary) !important;
+}
+
+/* About page */
+
+.about-list .wp-block-column:first-child {
+  padding-left: var(--spacing-14);
```
Comment (martinosmartinos) [inline css/style.css:2605]: Do we have any element with class 'about-list' right now?

File: css/style.css [hunk 2]
```diff
@@ -2587,6 +2594,39 @@ a.next.page-numbers:before {
...
+  left: 0;
+  font-size: var(--font-m);
+  margin-top: var(--spacing-6);
+}
+
+.dark-mode .about-list h1:before {
+  color: var(--theme-secondary);
+}
+
+.rhett-block h3 {
```
Comment (martinosmartinos) [inline css/style.css:2626]: Same comment as above: I do not see any element with this selector.

File: css/style.css (hunk 1 above)
Comment (Right-Noe) [reply css/style.css:2605]: Yes, Martin (wp-block-group about-list pv--40 is-layout-constrained wp-block-group-is-layout-constrained)

Comment (martinosmartinos) [reply css/style.css:2605]: Noelia, I don't get it as there is already page I created: https://news.mongabaydev.wpengine.com/about and it's linked in relevant places as well as placed in testing column in Asana. There is no need duplicating it which is why extra classes confused me.

**Review by martinosmartinos — APPROVED** [review]

**Pull request merged**